src/components/PostComment.jsx

```diff
@@ -0,0 +1,55 @@
+import { postComments } from "../../api";
+import { useState } from 'react';
+
+const PostComment = ({article_id}) => {
+    const [loading, setLoading] = useState(false)
+    const [error, setError] = useState(null)
+    const [commentBody, setCommentBody] = useState("")
+    const [username, setUsername] = useState("")
+
+
+const handleUsernameChange = (event) => {
+    setUsername(event.target.value)
+}
+ 
+const handleCommentChange = (event) => {
+    setCommentBody(event.target.value)
+}
+
+const handlePostComment = () => {
+        setLoading(true)
+        postComments(article_id, username, commentBody)
+        .then((response) => {
+            console.log("your comment has successfully been posted")
```

Comment:
you'll want to remove this `console.log`. Would you want to update the UI to inform that it has been posted, or is it obvious enough already?

Reply:
I've just checked and you're not updating the UI in anyway. You'll need to let to person see that something been posted, and they'll likely want to see that thing they have posted.